docs/self-managed/setup/guides/connect-to-an-oidc-provider.md

```diff
@@ -34,11 +34,12 @@ configuration](#component-specific-configuration) to ensure the components are c
 <h3>Steps</h3>
 
 1. In your OIDC provider, create an application for each of the components you want to connect. The expected redirect URI of the component you are configuring an app for can be found in [component-specific configuration](#component-specific-configuration).
-2. Make a note of the following values for each application you create:
+2. For all components except Web Modeler, the application has to be of confidential access type. For Web Modeler, which is a single page web application, the application has to be of public access type.
```

Comment:
To make this additionally generic, would the following be appropriate?:

```suggestion
2. For all Components, ensure the appropriate application type is used:
   - **Operate, Tasklist, Optimize, Identity:** Web apps requiring confidential access/a confidential client
   - **Modeler, Console:** Single-page applications requiring public access/a public client
except Web Modeler, ensure the application is confidential. For Web Modeler, which is a single page web application, the application has to be of public access type.
```

I've added Console here as it's listed as a single-page app so I expect it would have the same requirements as Web Modeler - do you know if this assumption is correct? I think this bridges the gap between the original Keycloak ask and also remaining generic for other OIDC providers which might require a version of this information.

Reply:
Hi @conceptualshark, I haven't used Console in my self-managed setup so far, but it is mentioned in the MS Entra section. It should be correct to add it here.
Your change improved it a lot, let's go to master!
Thank you!